lib/theme-config.js

```diff
@@ -121,6 +121,15 @@ ThemeConfig.prototype.setThemePath = function (themePath) {
     return this;
 };
 
+/**
+ * Reset Current Variation Settings
+ *
+ * @returns {ThemeConfig}
+ */
+ThemeConfig.prototype.resetVariationSettings = function () {
+    this.currentVariationSettings = null;
+};
+
```

Comment:
do we need to return something here ?

Reply:
Nope. It's just a setter. Some methods here return `this` for chaining, but I haven't found any occurrences of this.